### ISSUE TITLE
Allow product description to show images in the shopfront

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#6219]
## Description
**- As a:** shopper
**- On page:** product description in the shopfront
**- I want to be able to do:** see images

Currently the backoffice enables admin to had images, but the shopfront does not show them. Example on BE prod:

![image](https://user-images.githubusercontent.com/6525576/96420033-422f7c00-11f5-11eb-9482-6a300b339446.png)

## Acceptance Criteria & Tests
1. Log in as admin and go to `admin/products`
2. Create a product or edit a product
3. Add images in the description
4. Add this product to an open OC
5. Go visit the shopfront and open the product description page.
6. See the images you have added in the back office :+1:

**Comment by Erioldoesdesign** [comment]
So you mean that you should be able to, as a user filling out a CSV file add a URL for an image like this:
![Screenshot 2020-11-03 at 16 28 04](https://user-images.githubusercontent.com/11681324/98013935-ebc85d00-1df2-11eb-8371-c380c23da03e.png)

And have it display like this in the BO:
<img width="992" alt="Screenshot 2020-11-03 at 16 38 09" src="https://user-images.githubusercontent.com/11681324/98013990-fc78d300-1df2-11eb-98e5-ab525e9124d7.png">

And then like this in the shopfront:
<img width="944" alt="Screenshot 2020-11-03 at 16 38 38" src="https://user-images.githubusercontent.com/11681324/98014046-0bf81c00-1df3-11eb-800b-aac399859442.png">


I tried this 'add a URL for an image' in the CSV and yes it doesn't come through as an image. I imagine the CSV file would need a new column to parse an image. but I'll wait until I understand because in your example, there are small allergy images in the description and that might be the image you refer to, not the product image.

**Comment by RachL** [comment]
@Erioldoesdesign yes the small allergy images. You will need to insert html in the csv file to make it work in the admin, like:

```
<img
     src="image slug"
     alt="allergy logos"
/>
```

**Comment by Erioldoesdesign** [comment]
Ah yes okay makes sense 👍 

I wonder if it's too much to ask for this section on the right of the import screen to be updated with how you can add images in the description section using HTML:
<img width="492" alt="Screenshot 2020-11-03 at 17 13 33" src="https://user-images.githubusercontent.com/11681324/98018045-ffc28d80-1df7-11eb-8ad9-ec02bd0126e1.png">

At some future point, some examples in the CSV download template might be helpful too! (but that is certainly out of scope for this papercut 😅 )

**Comment by RachL** [comment]
@Erioldoesdesign sure and it is not just about image but any markup: bold, italic, space... for any visual effect to work during import, it must be done through html.
